non_atomic_component_message_processing/section_subject_domain_of_dialog_system_messages/subject_domain_of_phrases/concepts/concept_declarative_phrase/concept_error_phrase/concept_error_phrase.scs

```diff
@@ -0,0 +1,45 @@
+concept_error_phrase
+<- sc_node_class;
+=> nrel_main_idtf:
+    [фраза ошибки]
+    (* <- lang_ru;; *);
+    [error phrase]
+    (* <- lang_en;; *);
+    [Phrase über den Fehler]
+    (* <- lang_de;; *);
+<- rrel_key_sc_element:
+    ...
+    (*
+    => nrel_main_idtf:
+        [Опр. (фраза ошибки)]
+        (* <- lang_ru;; *);
+        [Def. (error phrase)]
+        (* <- lang_en;; *);
+        [Phrase über den Fehler]
+        (* <- lang_de;; *);;
+    <- definition;;
+    <= nrel_sc_text_translation:
+        ...
+        (*
+        -> rrel_example:
+            [Фраза ошибки - фраза при возникновении ошибки в системе.]
+                (* <- lang_ru;; *);;
+        -> rrel_example:
+            [Error phrase is a phrase about the occurrence of an error in the system.]
+            (* <- lang_en;; *);;
+
+        -> rrel_example:
+            [Phrase über den Fehler ist die Phrase über das Auftreten eines Fehlers im System.]
+            (* <- lang_de;; *);;
+        *);;
+    <= nrel_using_constants:
+        {
+            concept_phrase
+        };;
+    *);
+=> nrel_inclusion:
+    concept_answer_generation_error_phrase;
```

Comment:
cyclic inclusion

Reply:
sorry, don't see it

concept_error_phrase includes concept_answer_generation_error_phrase, looks like nothing wrong with it?